### PR TITLE
Updated styles for Write Comment

### PR DESCRIPTION
**Pull request by xtine**
Getting Write Mode closer to mockups:

nc-ss

### CODE REVIEW

File: regulations/templates/regulations/preamble-partial.html [hunk 1]
```diff
@@ -32,10 +32,12 @@
...
-    <div class="comment-index-item" data-comment-section="<%= comment.section %>">
-      <span><%= comment.section %></span>
-      <span class="comment-index-edit">Edit</span>
-      <span class="comment-index-clear">Clear</span>
-    </div>
+    <li class="comment-index-item group" data-comment-section="<%= comment.section %>">
+      <div class="comment-index-section"><%= comment.section %></div>
+      <div class="comment-index-modify">
+        <span class="comment-index-edit">edit</span>
+        <span class="comment-index-clear">x</span>
```
Comment (cmc333333) [inline regulations/templates/regulations/preamble-partial.html]: Probably a good idea to replace this with an icon (there's one in our font for "delete", I think) along with the "close" text for accessibility.

File: regulations/static/regulations/css/less/module/comment.less [hunk 2]
```diff
@@ -4,12 +4,20 @@ comment.less
 comment.less contains styles related to commenting on sections
 */
 
+@comment_blue: #4A90E2;
+@comment_blue_dark: #00629f;
+@comment_blue_light: #BDDCFD;
+@comment_item_blue: #E2F3F8;
+@comment_item_gray: #AEB0B5;
+@comment_input_border: #E5E5E5;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment.less]: This is much better than having the colors throughout, but we should aim to reuse variables which already exist in our pallet. We don't have a semantic pallet at the moment, but you could imagine this instead saying something like:
```less
@comment_active = @secondary_accent;
```

**Comment by cmc333333** [comment]
Two small speculations -- they shouldn't hold this back unless @xtine wants to take a stab at resolving.

File: regulations/templates/regulations/preamble-partial.html (hunk 1 above)
Comment (xtine) [inline regulations/templates/regulations/preamble-partial.html]: Both are going to be replaced by icons, but just waiting on final design approvals. Going off mock demos so setting up structure for now.

File: regulations/static/regulations/css/less/module/comment.less (hunk 2 above)
Comment (xtine) [inline regulations/static/regulations/css/less/module/comment.less]: Yeah at the moment don't know if the colors are final or not, will def later replace variables if they are repeated by the ones already defined in `variables.less` and make better names for them once things are more finalized.

**Pull request merged**